source/resource/models/entity/1.16.0/model_entity.json

```diff
@@ -56,7 +56,7 @@
               "identifier": {
                 "type": "string",
                 "description": "Entity definition and Client Block definition files refer to this geometry via this identifier.",
-                "pattern": "^geometry\\.[a-zA-Z0-9_\\-\\.]+$",
+                "pattern": "^geometry[\.:a-zA-Z0-9_\\-\\.]+$",

```

Comment:
Think you still wanna keep the dot right 🤔 

as it should be `geometry.`

Reply:
Ah right, also do you know why the checks failed?

Reply:
Hmm, Seems to be failing because of problems with the JSON structure 🤔 in the files you edited

Reply:
They're still valid json no?